services/evaluator/tests/integration/test_openai_specific.py

```
@@ -0,0 +1,190 @@
+Candidate: I have 3 years of Python experience with Django and Flask.
+Interviewer: How do you handle database queries?
+Candidate: I use Django ORM mostly, and sometimes raw SQL for complex queries.
+    """.strip()
+    
+    print(f"Prompt: {simple_prompt}")
+    print(f"Transcript length: {len(simple_transcript)} chars")
+    
+    try:
+        result = await call_openai_gpt5(simple_prompt, simple_rubric, simple_transcript)
+        
+        print(f"\n=== OpenAI Response ===")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)